FluentUI/build.gradle

```diff
@@ -97,9 +64,17 @@ publishing {
                 configurations.implementation.allDependencies.each {
                     if (it.group != null && (it.name != null && it.name != "unspecified") && it.version != null) {
                         def dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', it.group)
-                        dependencyNode.appendNode('artifactId', it.name)
-                        dependencyNode.appendNode('version', it.version)
+                        if (it.group.equals('fluentui-android')) {
```

Comment:
Please explain this if else
You can mention it in the comments

Reply:
Will explain it here instead of comments as this code is present in every module.

This code is to generate the pom file for bintray. When reading through the dependency list, whenever a project dependency is encountered, e.g 

implementation project(':fluentui_core')

The groupId is the project name i.e fluentui-android, hence for such cases we explicitly make groupid as com.microsoft.fluentui and read the version id from config.gradle